pkg/pod/pods.go

```diff
@@ -971,10 +971,18 @@ func (p *Pod) AppImageManifest(appName string) (*schema.ImageManifest, error) {
 // CreationTime returns the time when the pod was created.
 // This happens at prepare time.
 func (p *Pod) CreationTime() (time.Time, error) {
-	if p.isPrepared || p.isRunning() || p.AfterRun() {
-		return p.getModTime("pod")
+	if !(p.isPrepared || p.isRunning() || p.AfterRun()) {
+		return time.Time{}, nil
+	}
+	t, err := p.getModTime("pod-created")
```

Comment:
fly also creates it, but I understand this is another stage1 detail which may change at some point, and the ctime may not accurately reflect pod creation. Let's go for this.